nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-cluster/src/test/java/org/apache/nifi/cluster/integration/Cluster.java

```diff
@@ -131,7 +130,11 @@ public Node createNode() {
 
         final NiFiProperties nifiProperties = NiFiProperties.createBasicNiFiProperties("src/test/resources/conf/nifi.properties", addProps);
 
```

Comment:
Would there be a way to have an encryptor builder or factory that could just build this default method?

Reply:
I think that should be addressed through a different Jira. The deprecation of the initial method was done in NIFI-3116 and followed up by NIFI-4608.

Reply:
Thanks for the info @alopresto, I see there there was a utility function of some sort.  Why was it removed?  Would putting a builder or factory/default be a bad idea?